firebase-vscode/src/core/index.ts

```diff
@@ -0,0 +1,38 @@
+import vscode, { Disposable } from "vscode";
+import { ExtensionBrokerImpl } from "../extension-broker";
+import { registerConfig } from "./config";
+import { registerEmulators } from "./emulators";
```

Comment:
should cli be in core as well?

Reply:
arguably in utils, but if we do move this, it'll be a later CL, trying to keep these changes small since we don't have test coverage